wifiphisher/common/phishinghttp.py

```diff
@@ -19,13 +20,24 @@ def get(self, url):
         self.redirect("http://10.0.0.1:8080/")
 
 
-class ShowPhishingPageHandler(tornado.web.RequestHandler):
+class CaptivePortalHandler(tornado.web.RequestHandler):
 
     def get(self, url):
+        client_request = self.request.path[1:]
         try:
-            if "/" in url:
-                self.redirect("/index")
-            self.render("index.html", **template.get_context())
+            if self.request.path == "/":
```

Comment:
So here is my understanding of how it should work:
```
if user's requested page exists:
    display it to the user
otherwise:
    if any index file is found:
        display it to the user
    otherwise:
        throw an error 
```

Isn't there any way that we could use a wildcard here have something like this:
```
if "index.*" exists:
    do something
```

so we are no longer tied to the format?